src/ParameterLists.f90

```diff
@@ -2454,13 +2455,25 @@ RECURSIVE FUNCTION validateReq_ParamType(thisParam,reqParams,prefix,validType,is
               SELECTCASE(validType)
                 CASE(VALIDTYPE_VERIFYLIST,VALIDTYPE_VERIFYTEST)
                   isMatch=.FALSE.
-                  CALL eParams%raiseError(modName//'::'//myName// &
-                      ' - When verifying that parameters are equal, the parameter "'// &
-                      prefix//p%name//'" was not found on both lists!')
+                  IF(PRESENT(e)) THEN
+                    CALL e%raiseError(modName//'::'//myName// &
+                        ' - When verifying that parameters are equal, the parameter "'// &
+                        prefix//p%name//'" was not found on both lists!')
+                  ELSE
+                    CALL eParams%raiseError(modName//'::'//myName// &
```

Comment:
So it is ultimately because the `validateReq` subroutine is used in various fashion by 3 different wrapping subroutines.  One validates the parameter lists like how we do for the various lists setup in the Reactor subpackage.  One is for verifying lists in unit tests, and this one is for verifying lists in executable code.  I suppose I can enforce the exception handler to be passed through the `validateReq` calls and just have it pass in `eParams` for those other two wrapping subroutines I mentioned.  I'll do that a second.